helmfile/upstream/prometheus-elasticsearch-exporter/values.yaml

```diff
@@ -74,7 +86,6 @@ deployment:
 ##   KEY_2: value2
 env: {}
```

Comment:
I think they use a different method to connect to opensearch now, if you check the logs or the grafana dashboard you should see that there are no metrics